amenities.mss

```diff
@@ -321,6 +321,41 @@
     marker-clip: false;
   }
 
+  [feature = 'amenity_cinema'][zoom >= 17] {
```

Comment:
I would only show these extra POIs from zoom 18. What do you think?

Reply:
z17 because most of the usual amenities (shops...) begin at z17

Reply:
I would rather have them show only from Z18 as the map is already very heavily loaded at Z17 and these amenities are quite useless for a bike (except sometimes for the final destination).

Reply:
heavily loaded in Paris because it’s Paris, but in Bourgoins Jailleu not that much ;)
https://www.cyclosm.org/#map=17/45.58638/5.27592/cyclosm